tests/platform_tests/sfp/conftest.py

```diff
@@ -13,16 +12,9 @@ def teardown_module():
 
 
 @pytest.fixture(autouse=True)
-def disable_analyzer_for_mellanox(duthost):
+def update_la_ignore_errors_list_for_mlnx(duthost, loganalyzer):
     if duthost.facts["asic_type"] in ["mellanox"]:
-        loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix='sfp_cfg')
-        loganalyzer.load_common_config()
-
-        loganalyzer.ignore_regex.append("kernel.*Eeprom query failed*")
-        # Ignore PMPE error https://github.com/Azure/sonic-buildimage/issues/7163
-        loganalyzer.ignore_regex.append(r".*ERR pmon#xcvrd: Receive PMPE error event on module.*")
-        marker = loganalyzer.init()
-    yield
-
-    if duthost.facts["asic_type"] in ["mellanox"]:
-        loganalyzer.analyze(marker)
+        for host in loganalyzer:
+            loganalyzer[host].ignore_regex.append("kernel.*Eeprom query failed*")
+            # Ignore PMPE error https://github.com/Azure/sonic-buildimage/issues/7163
+            loganalyzer[host].ignore_regex.append(r".*ERR pmon#xcvrd: Receive PMPE error event on module.*")
```

Comment:
I think the newly added ignore error msg should be removed after the test, because we don't expect these error msg will be ignored in other tests. maybe you should add a yield section to remove them?:

```
    yield

    if duthost.facts["asic_type"] in ["mellanox"]:
         remove the newly added ignore msg
```

Reply:
Hi @keboliu 
Added fix as you suggested